lint(sh): add comments to to make shellcheck ignore doc strings (SC2016)

1. Clone and enter this repo:
   ```sh
   git clone https://github.com/webinstall/webi-installers.git ./webi-installers/
   cd ./webi-installers/
   ```
2. Install [`shellcheck`](https://webinstall.dev/shellcheck) and [`shfmt`](https://webinstall.dev/shfmt):
   ```sh
   curl -sS https://webinstall.dev/shellcheck | sh
   curl -sS https://webinstall.dev/shfmt | sh
   ```
3. FOLLOW THE ON-SCREEN INSTRUCTIONS (from the installs)
4. Check to see which scripts have docs that look like commands
   ```sh
   shellcheck -s sh -i SC2016 */*.sh */*/*.sh
   ```
   Example:
   ```txt
   In pyenv/install.sh line 12:
           echo 'eval "$(pyenv init -)"' >> ~/.bashrc
                ^----------------------^ SC2016 (info): Expressions don't expand in single quotes, use double quotes for that.

   In pyenv/install.sh line 13:
           echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
                ^-- SC2016 (info): Expressions don't expand in single quotes, use double quotes for that.
   ```
5. Add `# shellcheck disable=2016` above any `echo` that's being used to output docs or literal commands rather than for variable expansion. For example:
   ```diff
   +        # shellcheck disable=2016
            echo 'eval "$(pyenv init -)"' >> ~/.bashrc
   +        # shellcheck disable=2016
            echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
   ```
   (probably all of those shellcheck complaints should be ignored)

--- a/brew/install.sh
+++ b/brew/install.sh
@@ -37,6 +37,7 @@ _install_brew() {
         echo ""
         echo "If you prefer to have brew installed to '/usr/local' cancel now and do the following:"
         echo "        rm -rf '$HOME/.local/opt/brew'"
+        # shellcheck disable=2016
         echo '        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"'
         echo ""
         sleep 3
@@ -56,6 +57,7 @@ _install_brew() {
     echo ""
     echo "If you prefer to have brew installed to '/usr/local' do the following:"
     echo "        mv '$HOME/.local/opt/brew' '$HOME/.local/opt/brew.$(date '+%F_%H-%M-%S').bak'"
+    # shellcheck disable=2016
     echo '        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"'
     echo ""
 }

--- a/pyenv/install.sh
+++ b/pyenv/install.sh
@@ -9,7 +9,9 @@ __init_pyenv() {
     if [ ! -f ~/.bashrc ] || ! grep -q 'pyenv init' ~/.bashrc; then
         echo '' >> ~/.bashrc
         echo '# added by Webi for pyenv' >> ~/.bashrc
+        # shellcheck disable=2016
         echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+        # shellcheck disable=2016
         echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
     fi
 
@@ -18,7 +20,9 @@ __init_pyenv() {
         if ! grep -q 'pyenv init' ~/.zshrc; then
             echo '' >> ~/.zshrc
             echo '# added by Webi for pyenv' >> ~/.zshrc
+            # shellcheck disable=2016
             echo 'eval "$(pyenv init -)"' >> ~/.zshrc
+            # shellcheck disable=2016
             echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.zshrc
         fi
     fi

--- a/vps-addswap/install.sh
+++ b/vps-addswap/install.sh
@@ -9,6 +9,7 @@ __init_vps_addswap() {
 
     if [ "$(id -u)" -ne 0 ]; then
         echo Usage:
+        # shellcheck disable=2016
         echo '        sudo env PATH="$PATH" vps-addswap' "$my_size"
         exit 1
     fi

